leaderboard/fxa/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from leaderboard.fxa.views import FXARedirectView
+
+urlpatterns = [
+    url('^redirect/', FXARedirectView.as_view(),
+        name='fxa-redirect'),
```

Comment:
I would pull out 'fxa-redirect' as a constant as you're referencing it in reverse() function calls.